script/RegisterL1GovernorMetadataBridge.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Script, stdJson} from "forge-std/Script.sol";
+
+import {L2GovernorMetadata} from "src/L2GovernorMetadata.sol";
+import {Constants} from "test/Constants.sol";
+
+/// @dev This script will register the chain and address of the L1 metadata contract on the L2
```

Comment:
Am I correct in saying this must be run by the `owner` address? Might just be good to note this in the comments.

Reply:
Yes, that is correct and I will add that

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/8/commits/e6d6ab6e53a69f70e9bb63ccba2ef916d186f9bf